### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by jchave37**

### CODE REVIEW

File: src/test/java/com/revature/testing/BatchConstraintTest.java [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+
+
+
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+
+import org.junit.Before;
+
```
[the remark this answers is not in the document]
Comment (mgoshorn) [reply src/test/java/com/revature/testing/BatchConstraintTest.java]: @Hayden351

**Comment by Hayden351** [comment]
Added newest version of the validator to the Batch microservice.

**Comment by jchave37** [comment]
Removed not nulls validations for certain fields in Batch bean

**Comment by mgoshorn** [comment]
Still needing some confirmation that changes from Date to LocalDate will not break the handling on the Angular side. Test the values and if necessary create a custom serializer to write the format of the response in a way that the Angular application understands.

**Comment by mgoshorn** [comment]
@jchave37 @Hayden351 @jacastano 
These merge conflicts still need to be resolved.

**Pull request merged**